internal/integration/cli/pki.go

```diff
@@ -0,0 +1,70 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package cli
+
+import (
+	_ "embed"
+	"os"
+	"path/filepath"
+)
+
+var (
+	//go:embed "testdata/pki/ca.crt"
+	pkiCACrt []byte
+	//go:embed "testdata/pki/ca.key"
+	pkiCAKey []byte
+	//go:embed "testdata/pki/front-proxy-ca.crt"
+	pkiFrontProxyCACrt []byte
+	//go:embed "testdata/pki/front-proxy-ca.key"
+	pkiFrontProxyCAKey []byte
+	//go:embed "testdata/pki/sa.key"
+	pkiSAKey []byte
+	//go:embed "testdata/pki/etcd/ca.crt"
+	pkiEtcdCACrt []byte
+	//go:embed "testdata/pki/etcd/ca.key"
+	pkiEtcdCAKey []byte
+)
+
+func createTempPKIDir() (string, error) {
+	dir, err := os.MkdirTemp("", "talos-gen-test-")
```

Comment:
we can do `suite.T().TempDir()` which has automatic cleanup

Reply:
done